tests/integration/fixtures/bacnet-network/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04 as base
```

Comment:
Should we update to 22.04, which is GitHub actions `ubuntu-latest`?

Reply:
Yep!